make/test.mk

```diff
@@ -345,3 +355,39 @@ CLEAN_TARGETS += clean-coverage-unit
 # Removes unit test coverage file
 clean-coverage-unit:
 	-@rm -f $(COV_PATH_UNIT)
+
+#-------------------------------------------------------------------------------
+# e2e test in dev mode
+#-------------------------------------------------------------------------------
+
+.PHONY: build-image-local
+build-image-local: e2e-setup
+	eval $$(minishift docker-env) && operator-sdk build $(shell minishift openshift registry)/$(TEST_NAMESPACE)/devopsconsole-operator
+
+.PHONY: e2e-local
+e2e-local: build-image-local
+	@-oc login -u system:admin
+	@-oc project $(TEST_NAMESPACE)
+	@-oc create -f $(CUR_DIR)/deploy/crds/devopsconsole_v1alpha1_component_crd.yaml
+	@-oc create -f $(CUR_DIR)/deploy/service_account.yaml --namespace $(TEST_NAMESPACE)
+	@-oc create -f $(CUR_DIR)/deploy/role.yaml --namespace $(TEST_NAMESPACE)
+ifeq ($(UNAME_S),Darwin)
+	@sed -i "" 's|REPLACE_NAMESPACE|$(TEST_NAMESPACE)|g' $(CUR_DIR)/deploy/test/role_binding_test.yaml
+else
+	@sed -i 's|REPLACE_NAMESPACE|$(TEST_NAMESPACE)|g' $(CUR_DIR)/deploy/test/role_binding_test.yaml
```

Comment:
@tinakurian this syntax is wont fit in macOS env there fore my contribution here
01d3344 to make it work on mac.

Reply:
Corrine read my mind about being able to run on Mac or Linux - approved. 

(Thx to Corrine for additional test ideas too!)